apps/tor-socks-proxy/docker-compose.yml

```
@@ -0,0 +1,12 @@
+version: "3.9"
+
+services:
+  tor-socks-proxy:
+    container_name: tor-socks-proxy
+    image: peterdavehello/tor-socks-proxy:latest
+    restart: unless-stopped
+    ports:
+      - ${APP_PORT}:9150/tcp
+    networks:
+      - tipi_main_network
+    
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)